components/automate-cli/cmd/chef-automate/cliUtils.go

```diff
@@ -49,14 +49,17 @@ func RunCmdOnSingleAutomateNode(cmd *cobra.Command, args []string) error {
 
 	output, err := sshUtil.connectAndExecuteCommandOnRemote(script, true)
 	if err != nil {
+		if len(output) != 0 {
```

Comment:
Trim the output for space and new line before checking length.

Reply:
done